spec/classes/snmp_init_spec.rb

```diff
@@ -623,7 +623,7 @@
         }
 
         case facts[:os]['release']['major']
-        when '8', '9', '10', '11', '16.04', '18.04', '20.04'
+        when '8', '9', '10', '16.04', '18.04', '20.04', '22.04'
```

Comment:
Looks like you removed the `11` by mistake :wink:

Reply:
Gah, you're right, it's been one of those weeks... Fixed and pushed